packages/itmat-interface/test/serverTests/study.test.ts

```diff
@@ -927,7 +937,7 @@ describe('STUDY API', () => {
                 expect(res.status).toBe(200);
                 expect(res.body.errors).toBeUndefined();
                 createdUserAuthorised = await mongoClient.collection(config.database.collections.users_collection).findOne({ username });
-                expect(res.body.data.createUser).toEqual({
+                expect(res.body.data.createUser).toMatchObject({
```

Comment:
Because the values of $createdAt and $expiredAt are set to the instant time, i,e., dynamically. Thus I cannot know the exactly value of it.

Reply:
@wsy19961129 
mock the Date.now() function like this:
[https://github.com/facebook/jest/issues/2234](https://github.com/facebook/jest/issues/2234)

toMatchObject() is less strict than toEqual()
we do not want to compromise on that. it can miss future errors